react-app/src/api/slashingAPI.ts

```diff
@@ -20,10 +22,27 @@ export const useSlashingAPI = (): ISlashingAPI => {
     [query.slashing]
   );
 
+  /**
+   * get validator's uptime ratio since last unjail block
+   */
+  const getValidatorUptime = useCallback(
+    async (consensusAddress: string) => {
+      const signingInfo = await getSigningInfo(consensusAddress);
+      const height = await stargateQuery.getHeight();
+      return (
+        1 -
+        signingInfo.missedBlocksCounter.toNumber() /
+          Long.fromNumber(height).subtract(signingInfo.startHeight).toNumber()
```

Comment:
What are the differences between using Long and just regular number?

Reply:
its longer?